ams/ams-server/src/main/java/com/netease/arctic/ams/server/repair/command/ShowCall.java

```diff
@@ -1,42 +1,45 @@
 package com.netease.arctic.ams.server.repair.command;
 
 import com.netease.arctic.ams.server.repair.Context;
-import com.netease.arctic.catalog.ArcticCatalog;
 import com.netease.arctic.catalog.CatalogManager;
-import com.netease.arctic.table.TableIdentifier;
+import org.apache.commons.lang3.StringUtils;
 
 import java.util.stream.Collectors;
 
 public class ShowCall implements CallCommand {
 
-  private CatalogManager catalogManager;
   private Namespaces namespaces;
+  private CatalogManager catalogManager;
 
-  public ShowCall(CatalogManager catalogManager, Namespaces namespaces) {
-    this.catalogManager = catalogManager;
+  public ShowCall(String amsAddress, Namespaces namespaces) {
     this.namespaces = namespaces;
+    this.catalogManager = new CatalogManager(amsAddress);
```

Comment:
Maybe creating CatalogManager in Generator is better.

Reply:
@HuangFru I agree with you